### PR TITLE
Ws date picker preact

### PR DESCRIPTION
**Pull request by faboweb**
see #2 for an image of the new date picker

### CODE REVIEW

File: components/ws-date-picker/ws-date-picker-calendar-row.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		return momentDay.month() === expectedMonth;
+	}
+	isActive(year, month, date) {
+		return this.props.momentDay
+			&& year === this.props.momentDay.year()
+			&& month === this.props.momentDay.month()
+			&& date === this.props.momentDay.date();
+	}
+	render() {
+		let weeknum = this.props.weeknum;
```
Comment (fokusferit) [inline components/ws-date-picker/ws-date-picker-calendar-row.js]: is it maybe "weekNumber" ? to be more consistent here

Comment (fokusferit) [inline components/ws-date-picker/ws-date-picker-calendar-row.js]: And would use `let { weeknum, month , year} = this.props;`

File: package.json [hunk 2]
```diff
@@ -53,10 +53,12 @@
     "lite-server": "^2.2.0",
     "mocha": "3.2.0",
     "mocha-webpack": "0.7.0",
+    "moment": "2.0.0",
```
Comment (fokusferit) [inline package.json]: Just curious are we using here only moment or also moment.js with locales ?  As with locales it is quite 'big' (70kb~). But I think this is something we should consider for  #7 when building it.

Comment (fragsalat) [inline package.json]: I don't like 3rdParty dependencies but if this is working I'll accept this now and mark it as to be refactored.
Funny side fact that also moment introduces new complexity in the back^^

File: webpack.config.js [hunk 3]
```diff
@@ -7,24 +7,28 @@ module.exports = {
...
-        loader: 'sass-loader', // compiles Sass to CSS
-      }],
-    }],
+		rules: [{
+        test: /\.js$/,
+        use: [{
+          loader: 'babel-loader',
+          options: { presets: ['es2015'] },
+        }],
+				exclude: [/node_modules/]
```
Comment (fragsalat) [inline webpack.config.js]: Formatting sux but it's ok for now.

**Pull request merged**